components/Question.tsx

```diff
@@ -1,6 +1,7 @@
+// import { Select } from 'antd';
 import React from 'react';
-
-interface Option {
+// const { Option } = Select;
+interface OptionProps {
```

Comment:
```suggestion
interface Option {
```
naming this as `Option` is better